src/pages/my_page/MyPagePage.tsx

```diff
@@ -207,9 +208,10 @@ const MyPagePage = () => {
                 showCheckbox
                 isSelected={selectedTypes.publishingRequest}
                 color="publishingRequest"
-                onClick={() =>
-                  setSelectedTypes({ ...selectedTypes, publishingRequest: !selectedTypes.publishingRequest })
-                }>
+                onClick={() => {
+                  setSelectedTypes({ ...selectedTypes, publishingRequest: !selectedTypes.publishingRequest });
+                  deleteFromParameter(searchParams, navigate);
```

Comment:
Bare en liten notis om knappetrykk her nå vil føre til både oppdatering av state og en navigate. Det vil føre til (minst?) 2 renders, og jeg tenkte det ville føre til en feil, men ser ut som det virket fint. I så fall er det sikkert greit tenker jeg.

Alt annet er vel flyttet til URL params nå da, så selected types burde også det, siden det nå er det eneste som ikke tas vare på ved bokmerking. Egentlig up for grabs for hvem som helst å fikse det ved en ledig periode. Når vi får fiksa det vil vi unngå dobbel render her også

Reply:
Kan lage en oppgave på det 👍